### DISCUSSION TITLE
SO Upgrade Path

### DISCUSSION THREAD

**Discussion by udi-mosh** [Security-Onion-Solutions/securityonion#10248] [General]
Hi,
What is the correct upgrade path for SO ?
can i upgrade from 2.3.210 directly to 2.3.240 ?
is every ISO is a cumulative update for previous releases ?

**Comment by cm-ops** [comment]
Yes you can `soup` from `2.3.210` to `2.3.240` The upgrade will include the other release upgrades as well. Also, see https://docs.securityonion.net/en/latest/soup.html? for information on `soup`.

**Reply by udi-mosh** [reply]
Thank you. it worked...
